container/features/src/main/resources/features.xml

```diff
@@ -1440,14 +1440,13 @@
         <bundle>mvn:org.opennms/opennms-model/${project.version}</bundle>
     </feature>
 
-    <feature name="opennms-kvstore-api" description="OpenNMS :: Features :: Distributed :: Key Value Store :: API" version="${project.version}">
-        <bundle>mvn:org.opennms.features.distributed/org.opennms.features.distributed.kv-store.api/${project.version}</bundle>
+    <feature name="opennms-kvstore-shell" description="OpenNMS :: Features :: Distributed :: Key Value Store :: Shell" version="${project.version}">
         <bundle>mvn:org.opennms.features.distributed/org.opennms.features.distributed.kv-store.shell/${project.version}</bundle>
     </feature>
     
     <feature name="opennms-kvstore-noop"
              description="OpenNMS :: Features :: Distributed :: Key Value Store :: No-Op" version="${project.version}">
-        <feature>opennms-kvstore-api</feature>
+        <feature>opennms-kvstore-shell</feature>
```

Comment:
We normally depend directly on the api feature and not the shell.

I guess the shell bundle should always be loaded by the `opennms-kvstore-api` feature.

Reply:
I can't depend on the API here since it is being provided by the root class loader on OpenNMS. Only the sentinel version of the kvstore features are bringing along an api bundle since it won't be provided otherwise there.

Reply:
Ack. That make sense.